README.md

```diff
@@ -62,8 +63,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 > Below are important links to live project .
-- [Live Demo Link]()
-- [Loom Video Recording]()
+- [Live Demo Link](https://cointracker-smct.onrender.com/)
```

Comment:
**Well done!
You did a great job!**

- [ ] This is the only issue that I was able to find. In the testing part of your app, I can see only the components test. But, in the mock testing folder, I was not able to find any tests. Please add mock tests to your project and make sure they are passing as expected!

HINT: You can perform a mocking data fetching and Redux Store connection.

Reply:
Hello,
- Make the correction in the README.md file.
- Create coinsContaine.test.js file in the __test__ folder and it's passed successfully.